### PR TITLE
Multi race election

### PR DESCRIPTION
**Pull request by mikefranze**
Built on top of #171 
Adds multiple race support
Some of it can be cleaned up a bit more but good enough for first cut. Currently all ballots are displayed on one page, should probably only show one at a time with buttons to flip through them.
Doesn't include precinct logic yet, will tackle that next.
Demo: https://www.youtube.com/watch?v=Bwb6OHTzkpY

### CODE REVIEW

File: backend/src/Controllers/getElectionResultsController.ts [hunk 1]
```diff
@@ -22,31 +22,32 @@ const getElectionResults = async (req: any, res: any, next: any) => {
...
 
     const election = req.election
-    const candidateNames = election.races[0].candidates.map((Candidate: any) => (Candidate.candidate_name))
-    const cvr = ballots.map((ballot: Ballot) => (
-        ballot.votes[0].scores.map((score: Score) => (
-            score.score
+    let results = []
+    for (let race_index = 0; race_index < election.races.length; race_index++) {
+        const candidateNames = election.races[race_index].candidates.map((Candidate: any) => (Candidate.candidate_name))
+        const cvr = ballots.map((ballot: Ballot) => (
```
Comment (scottPlusPlus) [inline backend/src/Controllers/getElectionResultsController.ts:28]: what does cvr stand for?

**Review by scottPlusPlus — CHANGES_REQUESTED** [review]
Can we get some testing on the backend for multi-election races?

Also add some validation when a vote is submitted that it contains the expected number of races.

File: backend/src/Controllers/getElectionResultsController.ts (hunk 1 above)
Comment (mikefranze) [reply backend/src/Controllers/getElectionResultsController.ts:28]: Tests and validation added. 
CVR stands for cast vote record, was inherited from Jay's old code.

**Review by scottPlusPlus — APPROVED** [review]
awesome!

**Pull request merged**